lib/pages/timetable/model/events/uni_event.dart

```diff
@@ -173,3 +173,39 @@ class UniEventInstance extends Event {
   int get hashCode =>
       hashList([super.hashCode, color, location, mainEvent, title]);
 }
+
+extension UniEventInstanceExtension on UniEventInstance {
```

Comment:
I mean, since you're adding it to the same file anyway, I don't see a reason to use extension. Just add the methods to the class directly.